gdal/gdal_merge.xml

```diff
@@ -0,0 +1,211 @@
+        gdal_merge.py
+            -q        
+            #for $i, $s in enumerate( $series )
+                '${s.input}'
+            #end for
+            
+            -of '$of'
+            
+            #if $settings.advanced=='advanced'
+                
+                #if $settings.co!=''
+                    -co '$settings.co'
+                #end if
+                
+                #if $settings.ot!=''
+                    -ot '$settings.ot'
+                #end if
+                
+                #if $settings.condi_ps.pixelsize=='use_ps'
+                    -ps $settings.condi_ps.pixelsize_x $settings.condi_ps.pixelsize_y
+                #end if
+                
+                $settings.tap
+                
+                #if $settings.condi_ul_lr.ul_lr=='ul_lr'
+                    -ul_lr $settings.condi_ul_lr.ulx $settings.condi_ul_lr.uly $settings.condi_ul_lr.lrx $settings.condi_ul_lr.lry
+                #end if
+                
+                $settings.v
+                
+                $settings.separate
+                
+                $settings.pct
+                
+                #if $settings.n!='nodata_value'
+                    -n '$settings.n'
+                #end if
+                
+                #if $settings.a_nodata!='output_nodata_value'
+                    -a_nodata '$settings.a_nodata'
+                #end if
+                
+                #if $settings.init!=''
+                    -init '$settings.init'
+                #end if
+                
+            #end if
+            
+            -o '$output'
+        ]]></command>
+        
+    <inputs>
+        <repeat name="series" title="Input file">
+            <param type="data" name="input" format="data" label="Gdal supported input file" help="http://www.gdal.org/formats_list.html"/>
+        </repeat>
+        <expand macro="gdal_param_of"/>
+        <conditional name="settings">    
+            <expand macro="gdal_advanced_params_select"/>
+            <when value="advanced">
+                <expand macro="gdal_param_co"/> 
+                <expand macro="gdal_param_ot"/> 
+                <conditional name="condi_ps">
+                    <param label="-ps pixelsize_x pixelsize_y" name="pixelsize" type="boolean" truevalue="use_ps" falsevalue="no_ps" checked="false" />
+                    <when value="use_ps">
+                        <param name="pixelsize_x" type="integer" value="0" min="0"/>
+                        <param name="pixelsize_y" type="integer" value="0" min="0"/>
+                    </when>
+                    <when value="no_ps">
+                    </when>
+                </conditional>        
+                <expand macro="gdal_param_tap"/> 
+                <conditional name="condi_ul_lr">
+                    <param name="ul_lr" type="select" label="-ul_lr ulx uly lrx lry" >
+                        <option value="no_ul_lr" selected="true">Don't use the -ul_lr option</option>
```

Comment:
Can we enhance the label here? Do people know what the ul_lr option is? What does it Do?

Reply:
second pass done